app/models/measurement.rb

```diff
@@ -1,3 +1,27 @@
 class Measurement < ApplicationRecord
-  belongs_to :tank
+  belongs_to :measurement_event
+
+  delegate :tank, to: :measurement_event
+  def self.create_from_csv_data(attrs)
+    # remove relational (non-attribute) data from hash to be handled separately
+    measurement_event_name = attrs.delete(:measurement_event)
+    tank = Tank.find_or_create_by!(name: attrs.delete(:name))
```

Comment:
These `find_or_create_by!` calls freak me out in several ways, but they'll make our lives way easier for the demo. 

Down the road we should
- add a nice way to manage tanks/measurement events and not allow them to be autocreated in the import
and/or
- add a way to merge tanks or measurement events together if names are similar but not exact.

But that's a problem for later 😄